pkg/datapath/link/link.go

```diff
@@ -92,16 +94,20 @@ func (c *ifNameCache) syncCache() error {
 	return nil
 }
 
-func (c *ifNameCache) lookupName(ifIndex int) (string, bool) {
+func (c *LinkCache) lookupName(ifIndex int) string {
 	c.mu.RLock()
 	defer c.mu.RUnlock()
 
 	name, ok := c.indexToName[ifIndex]
-	return name, ok
+	if !ok {
+		name = fmt.Sprintf("%d", ifIndex)
+	}
+	return name
```

Comment:
This looks like an innocent change but this might have performance implications in hubble which uses this function. I suggest to replace the `fmt.Sprintf` with a faster function. Probably `strconv.Itoa`.

Reply:
The code is already doing this today, but while we're here we may as well fix this up yeah :+1: 

No idea how accurate this is but here's some random gist about performance of these alternatives:
https://gist.github.com/evalphobia/caee1602969a640a4530